src/rpc/handlers/AccountTx.cpp

```diff
@@ -180,12 +191,16 @@ doAccountTx(Context const& context)
             obj[JS(date)] = txnPlusMeta.date;
         }
         obj[JS(validated)] = true;
-
         txns.push_back(obj);
-        if (!minReturnedIndex || txnPlusMeta.ledgerSequence < *minReturnedIndex)
+
+        if (!minReturnedIndex)
```

Comment:
I don't think this change is correct. We need to make sure minReturnedIndex is the min we see during the entire iteration. We need to check it every time, to see if it should be updated.

Reply:
@cjcobb23 will change it, but do we want similar logic for L202 for maxReturnedIndex (basically restoring txnPlusMeta.ledgerSequence > *maxReturnedIndex)